libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1848,7 +1848,7 @@ bool TypeChecker::visit(FunctionCall const& _functionCall)
 		if (functionType->takesArbitraryParameters())
 			m_errorReporter.typeError(
 				_functionCall.location(),
-				"Named arguments cannnot be used for functions that take arbitrary parameters."
+				"Named arguments cannot be used for functions that take arbitrary parameters."
```

Comment:
It seems there are no test cases for this, since no failure was triggered. Are you interested in creating a test case? It was not part of the original issue, but would be a good way to get started with tests if you are interested.

Reply:
Ok, this should be pretty straightforward. I will add the test to this PR as well.

Reply:
@axic I added a test case, passes locally. I hope this can be merged now together with codespell PR.